product_plan.md

```diff
@@ -0,0 +1,83 @@
+4. Distributed apps are too hard to develop and to be used by everyday people
+5. Issues involved in Proof of Work based consensus mechanisms, mining pools and ASIC mining
+6. Issues involved in Proof of Stake based consensus mechanisms
+
+## Overview
+
+Our short-term high-level product plan is to build [Spacemesh 0.1](spacemesh01.md), release testnets, follow up with `Spacemesh 0.x` updates and a launch a mainnet for this product according to the project [roadmap](roadmap.md).
+
+Our long-term high-level product plan is to implement, test and roll-out additional capabilities and features to the platform in subsequent incremental releases until the full platform is released as `Spacemesh 1.0` with support to all the main use cases outlined here.
+
+Incremental product releases, starting with `Spacemesh 0.1` will be deployed to testnets first for testing and to the `Spacemesh Mainnet` once they reach production quality status and have been security audited.
+
+## TL;DR
+
+![](https://raw.githubusercontent.com/spacemeshos/product/master/resources/roadmap2019.png)
+
+---
+
+## Spacemesh 0.1
+
+### Release Highlights
+
+The main use case case of `Spacemesh 0.1` is to support [Spacemesh Coin](spacemesh_coin.md) cryptocurrency transactions between any two parties and to award `Spacemesh coins` to people who run full Spacemesh full p2p nodes on their PCs. The Spacemesh full node implements the Spacemesh consensus protocol and other protocols to support this use case.
+
+The main goal of this release is to provide an MVP of a permissionless and trustless Internet money that is secure, decentralized and scalable without using POW or PoStake mechanisms.
+
+### Visual Story
+![Story](https://raw.githubusercontent.com/spacemeshos/product/master/resources/sm_0.1_story.jpg)
+
+### Release main software components
+
+### Use cases
+- [Available here](spacemesh01.md)
+
+### Main Components
+1. [go-spacemesh](https://github.com/spacemeshos/go-spacemesh) - The reference implementation of the Spacemesh full node in go-lang for Windows 10, OS X and Linux.
+
+2. [Spacemesh POET service](https://github.com/spacemeshos/POET) - a public utility used in the Spacemesh proofs of space time protocol
+
+3. [Spacemesh App](https://github.com/spacemeshos/smapp) - A desktop app for Windows 10, OS X and Linux which includes a Wallet, a managed full p2p node and mining.
+
+4. `Spacemesh Gateway` - Foundation deployed nodes providing the `Spacemesh API` for wallets and for the app to check coin balance, and execute transactions.
+
+5. [Spacemesh Tap](tap.md) - Enables users to get `Spacemesh Coins` on testnets for testing purposes.
+
+6. [Spacemesh POST](https://github.com/spacemeshos/post) - Implement the proofs of space protocol used by Spacemesh.
+
+For additional information about these components read this page in the [Spacemesh Testnet guide](https://testnet.spacemesh.io/#/dict)
+
+---
+
+## Spacemesh 0.x
```

Comment:
According to the table, this milestone will include "protocol updates" - which updates is included as part of this milestone?

Reply:
All the protocol updates that will come before the mainent and will need to be testend in testnet - mostly the open research issues that the research team is currently on that will need to be implemented before mainent.

Reply:
There is a list of 5-7 protocol updates (for e.g, sub ids, poet discovery, genesis flow, etc.) that I'm aware of, that require implementation, plus 15 more updates planned for mainnet that are pending heavy research. According to the product plan, all of those need to be delivered by Q1 2020 (4 months after testnet launch). 

Aviv,
Where can I find a detailed list of product requirements including the protocol requirements for release 0.x? 

@tal-m @iddo333, 
When should the dev team expect the full protocol specifications that we want to use for mainnet?